pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -148,6 +149,29 @@ func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
 	}
 }
 
+func loadConfigDir(dir string) []string {
+	stat, err := os.Lstat(RecommendedHomeFileDir)
```

Comment:
Perhaps skip if `dir == ""`, although I'm not entirely sure how PathOptions is generally used.